server/build-plugins.js

```diff
@@ -13,6 +13,8 @@ const logger = require('./logger');
 const config = new Config('bpanel');
 config.load({ env: true, argv: true, arg: true });
 const pluginsConfig = resolve(config.prefix, 'config.js');
+const modulesDirectory = resolve(__dirname, '../node_modules');
+const homePrefix = process.env.BPANEL_PREFIX;
```

Comment:
Is this a new environment variable being introduced?

Reply:
This was added in a previous branch where we refactored the configs but there wasn't an opportunity to really use it yet. It gets passed through at runtime from the server process that starts webpack [here](https://github.com/bpanel-org/bpanel/blob/development/server/index.js#L89) and then gets passed through to `build-plugins.js` in the webpack script [here](https://github.com/bpanel-org/bpanel/blob/development/webpack.config.js#L125).

Reply:
Could we add a default value for it just in case something goes wrong? Maybe a log line for what `homePrefix` is being used? If a major rearchitecuture somehow happens and that env var isn't set somehow, it could cause problems